ndarray/src/main/java/org/tensorflow/ndarray/Shape.java

```diff
@@ -177,6 +188,24 @@ public boolean isUnknown() {
     }
   }
 
+  /**
+   * Returns a defensive copy of the this Shape's axes. Changes to the returned list do not change this Shape's state.
+   * Returns null if {@link Shape#isUnknown()} is true.
```

Comment:
Why would we return null rather than `Collections.emptyList`?

Reply:
An empty shape is a valid shape though, for scalars.  Unknown is different.  Also I'm just matching `asArray`.

Reply:
Ok.